### PR TITLE
f-breadcrumbs@3.1.0 - Replace `<span>` with `<template>`

### PR DESCRIPTION
**Pull request by xander-marjoram**
### Changed
- Replace `<span>` with `<template>`.
- Simplify and shuffle around styles accordingly.

### Added
- Storybook control for `hasBackground`.

---

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/molecules/f-breadcrumbs/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
+v3.1.0
+------------------------------
+*October 27, 2021*
+
+### Changed
+- Replace `<span>` with `<template>`.
+- Simplify and shuffle around styles accordingly.
+
+### Added
+- Storybook control for `hasBackground`.
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-breadcrumbs/CHANGELOG.md:16]: Thanks for that 👍

File: packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue [hunk 2]
```diff
@@ -65,14 +57,15 @@ export default {
...
+        * @returns {*|string}
+        */
         linkActiveClass (index) {
-            return index === this.links.length - 1
-                ? this.$style['c-breadcrumbs-link--active']
-                : '';
+            const lastIndex = this.links.length - 1;
+            return index !== lastIndex ?
+                this.$style['c-breadcrumbs-link'] :
+                this.$style['c-breadcrumbs-link--active'];
```
Comment (DamianMullins) [inline packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue]: If this is a modifier does it also need the `c-breadcrumbs-link` class?

File: packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue [hunk 3]
```diff
@@ -123,30 +116,36 @@ $breadcrumbs-active-font-weight: $font-weight-regular;
...
 
         .c-breadcrumbs-list--hasBackground & {
             color: $breadcrumbs-text-colour-hasBackground;
         }
     }
+
+    a {
+        cursor: pointer;
+        text-decoration: none;
+        &:hover,
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue]: Could you line break above please.

File: packages/components/molecules/f-breadcrumbs/src/components/tests/Breadcrumbs.test.js [hunk 4]
```diff
@@ -142,74 +147,74 @@ describe('BreadCrumbs', () => {
...
 
                 // Assert
                 expect(breadcrumbLinks).toHaveLength(1);
-                expect(breadcrumbSpans).toHaveLength(1);
+                expect(breadcrumbElements).toHaveLength(3); // Includes separator
             });
         });
+    });
 
-        describe('`hasBackground`', () => {
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-breadcrumbs/src/components/tests/Breadcrumbs.test.js:153]: Not sure what happened here but, cheers.

File: packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue (hunk 2 above)
Comment (xander-marjoram) [reply packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue]: The way they are currently written, no, but that probably means I've written them incorrectly.

Would you prefer there to be a base class that gets applied to all links, and then (if necessary) two modifiers, i.e., `--active` and `--inactive`?

**Review by kevinrodrigues — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**